pkgs/tools/misc/nat/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nat";
+  version = "2.1.14";
+
+  src = fetchFromGitHub {
+    owner = "willdoescode";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256:0629nfcf43pzkx1rp6k1nr5sdpsjsgrs21f8yc7g20kxlnppc7z3";
+  };
+
+  cargoSha256 = "sha256:1kzsh8c1q51bfvpq13a1i74ss9yms4zjw8gxx2fggj69xhpy3pkg";
+
+  meta = with lib; {
+    description = "`ls` alternative with useful info and a splash of color art";
```

Maybe something like

```suggestion
    description = "Tool to list files or directories with color support";
```